1ejer/menu.css

```
@@ -0,0 +1,363 @@
+body {
+    margin: 0;
+}
+
+.menu-container {
+    border-bottom: 1px solid gray;
+}
+
+.menu {
+    margin: auto;
+  max-width: 1200px;
+  background-color: #ffffff;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  display: flex;
+ 
+}
+
+.logo img {
+  height: 47px;
+  margin: auto;
+  align-items: center;
+}
+
+.navigation {
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    padding-left: 0px;
+    align-items: center;
+
+}
+
+.addons {
+    list-style: none;
+    justify-content: space-between;
+    box-sizing: border-box;
+    align-items: center;
+    padding-left: 0;
+}
+
+.addons a {
+    width: 100%;
+    color: #222222;
+    font-family: 'Nunito Sans';
+    font-size: 14px;
+    font-weight: bold;
+    letter-spacing: 0;
+    line-height: 18px;
+    text-decoration: none;
+    text-transform: uppercase;
+    display: flex;
+    align-items: flex-start;
+}
+
+a {
+    text-decoration: none;
+}
+
+.live {
+    color: #ffffff;
+    font-family: 'Nunito Sans';
+    font-size: 9px;
+    font-weight: 800;
+    letter-spacing: 1.5px;
+    line-height: 12px;
+    text-align: center;
+    padding: 3px 6px;
+    border-radius: 3px;
+    background-color: #fb6a66;
+    box-shadow: 0 0 0 3px #ffe1e1;
+    margin-left: 9px;
+    margin-top: -1px;
```

Bien ahí loquilla. También podes usar 
```suggestion
    transform: translateY(-1px);
```